openmeetings-webservice/src/main/java/org/apache/openmeetings/webservice/UserWebService.java

```diff
@@ -156,7 +156,7 @@ public UserDTO add(
 			@WebParam(name="sid") @QueryParam("sid") String sid
 			, @WebParam(name="user") @FormParam("user") UserDTO user
 			, @WebParam(name="confirm") @FormParam("confirm") Boolean confirm
-			)
+			) throws ServiceException
```

Comment:
This one is unnecessary (and will produce Sonar error)
`ServiceException extends RuntimeException` so it shouldn't be explicitly declared

Reply:
It has to. That is the only way how it recognised by CXF to put mappings for this object into the response payload.
If you don't, then CXF (no matter if Soap or Rest) won't know what to do with this object and just return error 500 without any message.
See for example: https://stackoverflow.com/a/7992326/1448704

Reply:
this is unfortunate :(
In such case could you please do the following: 

- modify `ServiceException` so it would extend plain `Exception`
- add throws declaration where appropriate?